docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -353,115 +479,72 @@ export const pageQuery = graphql`
 
 Further examples can be found in the [Gatsby Image docs](https://github.com/gatsbyjs/gatsby/tree/d0e29272ed7b009dae18d35d41a45e700cdcab0d/packages/gatsby-image).
 
-## Manually specify PostCSS plugins
-
-Gatsby v2 removed `postcss-cssnext` and `postcss-import` from the default PostCSS setup.
-
-To have the same configuration that you had in v1, you should use [`gatsby-plugin-postcss`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-postcss) and follow the recommended migration path below.
+### Delete Nodes API Deprecated
 
-### 1. Install the dependencies
+Stub content. PRs welcome!
```

Comment:
`deleteNodes` is deprecated so instead you should write `nodes.forEach(n => deleteNode({node: n}))`